src/explorer/model/WorkspaceFolderNode.ts

```diff
@@ -39,4 +49,10 @@ export class WorkspaceFolderNode extends NodeBase {
     private async _searchForPomPaths(): Promise<void> {
         this._pomPaths = await Utils.getAllPomPaths(this._workspaceFolder);
     }
+
+    private async _sortChildren(): Promise<void> {
```

Comment:
I don't see any async things inside, so I think there's no need to declare as an `async` function.